src/Behat/Behat/Output/Node/Printer/Pretty/PrettySkippedStepPrinter.php

```diff
@@ -144,8 +145,10 @@ private function getArgumentString(ArgumentInterface $argument, $collapse = fals
 
             return $text;
         }
-
-        return (string) $argument;
+        if ($argument instanceof TableNode) {
+            return $argument;
```

Comment:
This should still cast the argument, otherwise it does not return a string.

Reply:
Updated